docs/mp/restclient/09_rest-client.adoc

```diff
@@ -34,69 +50,149 @@ include::{common-deps-page-prefix-inc}[tag=maven-dependency]
 </dependency>
 ----
 
-== Configuring Rest Client with Helidon MP
-MicroProfile Rest Client adds the capability to invoke remote microservices using a JAX-RS like interface to declare the
-operations.
+== API
+[cols="1,2"]
```

Comment:
Is this what we want to do? Seems like a link to javadocs would be a better fit.
Even if we do, the link should still be here (the class name should be a link), as that is the place I would expect to learn more